cloud/google/clusteractuator.go

```diff
@@ -67,3 +120,66 @@ func getOrNewComputeServiceForCluster(params ClusterActuatorParams) (GCEClientCo
 	}
 	return computeService, nil
 }
+
+func (gce *GCEClusterClient) createFirewallRuleIfNotExists(cluster *clusterv1.Cluster, firewallRule *compute.Firewall) error {
+	ruleExists, ok := cluster.ObjectMeta.Annotations[firewallRuleAnnotationPrefix + firewallRule.Name]
+	if ok && ruleExists == "true" {
+		// rule was already created, so just return
```

Comment:
use full sentences for comments
https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
done.